README.md

```diff
@@ -81,9 +81,11 @@ myResolver.resolve({}, lookupStartPath, request, resolveContext, (
 | alias            | []                          | A list of module alias configurations or an object which maps key to value                                                                    |
 | aliasFields      | []                          | A list of alias fields in description files                                                                                                   |
 | cacheWithContext | true                        | If unsafe cache is enabled, includes `request.context` in the cache key                                                                       |
+| conditionNames   | []                          | A list of exports field condition names                                                                                                       |
```

Comment:
enhanced-resolve acts as node.js resolver when used standalone. So here node.js defaults should be used (and in `lib/index.js`)

`["node"]`